.github/dependabot.yml

```
@@ -0,0 +1,82 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+    labels:
+      - "dependencies"
+    reviewers:
+      - "DaleStudy/daleui"
+    groups:
+      storybook:
+        patterns:
+          - "@storybook*"
+          - "storybook"
+          - "@chromatic-com/storybook"
+          - "chromatic"
+          - "eslint-plugin-storybook"
+        update-types:
+          - "minor"
+          - "patch"
+      react:
+        patterns:
+          - "react"
+          - "react-dom"
+          - "@types/react*"
+        update-types:
+          - "minor"
+          - "patch"
+      testing:
+        patterns:
+          - "@testing-library*"
+          - "vitest"
+          - "happy-dom"
+        update-types:
+          - "minor"
+          - "patch"
+      eslint:
+        patterns:
+          - "eslint*"
+          - "@eslint*"
+          - "typescript-eslint"
+        update-types:
+          - "minor"
+          - "patch"
+      typescript:
+        patterns:
+          - "typescript"
+          - "@types/*"
+        update-types:
+          - "minor"
+          - "patch"
+      vite:
+        patterns:
+          - "vite"
+          - "@vitejs/*"
+          - "vite-plugin-*"
+        update-types:
+          - "minor"
+          - "patch"
+      ui-components:
+        patterns:
+          - "@radix-ui/*"
+          - "lucide-react"
+        update-types:
+          - "minor"
+          - "patch"
+      css:
+        patterns:
+          - "@pandacss/*"
+        update-types:
+          - "minor"
+          - "patch"
+
+  - package-ecosystem: "github-actions"
+    directory: "/.github/workflows"
+    schedule:
+      interval: "monthly"
```

요것도 매일 체크로 함 가봅시다 ㅋㅋ

```suggestion
    schedule:
      interval: "daily"
```